analyzers/src/ITs.JsonParser/Json/RuleIssues.cs

```diff
@@ -30,16 +30,19 @@ public class RuleIssue
 {
     public string Id { get; set; }
     public string Message { get; set; }
-    public SarifLocationFile Location { get; set; } // TODO: This should not re-use SarifLocationFile type.
+    public string Uri { get; set; }
+    public string Location { get; set; }
 
     public RuleIssue(SarifIssue issue)
     {
         Id = issue.RuleId;
         Message = issue.Message;
-        Location = new()
+        Uri = issue.NormalizedUri();
+        if (issue.Location is { } location)
         {
-            Uri = issue.NormalizedUri(),
-            Region = issue.Location?.Region
-        };
+            Location = location.Region.StartLine == location.Region.EndLine
+                ? $"Line(s) {location.Region.StartLine} Position {location.Region.StartColumn}#{location.Region.EndColumn}"
```

Comment:
This doesn't need to be aligned with the next one, because they should not meet anywhere near each other
```suggestion
                ? $"Line {location.Region.StartLine} Position {location.Region.StartColumn}#{location.Region.EndColumn}"
```